krakenapi.go

```diff
@@ -197,6 +210,14 @@ func (api *KrakenAPI) OHLC(pair string) (*OHLCResponse, error) {
 	return ret, nil
 }
 
+// OHLC returns a OHLCResponse struct based on the given pair
+// Backward compatible with previous version
```

Comment:
Can you remove this line? No need to specifically point out the backwards compatibility.

Reply:
done.